rootfs/usr/share/www/index.html

```diff
@@ -1,169 +1,149 @@
-            fill="currentColor"
-            d="M8.59,16.58L13.17,12L8.59,7.41L10,6L16,12L10,18L8.59,16.58Z"
-          />
+            d="M3,4H20A2,2 0 0,1 22,6V8H18V6H5V18H14V20H3A2,2 0 0,1 1,18V6A2,2 0 0,1 3,4M17,10H23A1,1 0 0,1 24,11V21A1,1 0 0,1 23,22H17A1,1 0 0,1 16,21V11A1,1 0 0,1 17,10M18,12V19H22V12H18Z" />
         </svg>
+      </div>
+      Download our app
+    </div>
+  </div>
+
+  <div class="footer">
+    <a href="https://www.home-assistant.io/getting-started/onboarding/" target="_blank"
+      rel="noreferrer noopener">Help</a>
+  </div>
+
+  <dialog id="dialog-community">
+    <div class="headline">
+      <h1>Join our community</h1>
+      <form id="form" slot="content" method="dialog">
+        <button class="icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+            <path
+              d="M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z" />
+          </svg></button>
+      </form>
+    </div>
+    <a href="https://www.home-assistant.io/newsletter/" target="_blank" rel="noreferrer noopener" class="link">
+      <div>
+        <img loading="lazy" src="/landingpage/static/logo.svg" />
+        Home Assistant forums
+      </div>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+          d="M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z" />
+      </svg>
+    </a>
+    <a href="https://www.home-assistant.io/newsletter/" target="_blank" rel="noreferrer noopener" class="link">
+      <div>
+        <img loading="lazy" src="/landingpage/static/logo.svg" />
+        Building the Open Home newsletter
+      </div>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+          d="M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z" />
+      </svg>
+    </a>
+    <a href="https://www.home-assistant.io/newsletter/" target="_blank" rel="noreferrer noopener" class="link">
+      <div>
+        <img loading="lazy" src="/landingpage/static/logo_discord.png" />
+        Discord chat
+      </div>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+          d="M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z" />
+      </svg>
+    </a>
+    <a href="https://www.home-assistant.io/newsletter/" target="_blank" rel="noreferrer noopener" class="link">
+      <div>
+        <img loading="lazy" src="/landingpage/static/logo_twitter.png" />
+        Twitter
+      </div>
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <path
+          d="M14,3V5H17.59L7.76,14.83L9.17,16.24L19,6.41V10H21V3M19,19H5V5H12V3H5C3.89,3 3,3.9 3,5V19A2,2 0 0,0 5,21H19A2,2 0 0,0 21,19V12H19V19Z" />
+      </svg>
+    </a>
```

Comment:
Can we add top and bottom padding (8px) to match the onboarding style?
Also, the link are all the same.

Reply:
Where do you want this padding exactly?

Reply:
`mwc-list` has a 8px padding. I would wrap the link in a div and add `padding: 8px 0` to this div.